### ISSUE TITLE
StyledTextAreaBehaviorTest fail

### ISSUE THREAD

**Issue by msarmc** [TomasMikula/RichTextFX#454]
I'm trying to build the latest version and I get the following error.

org.fxmisc.richtext.model.StyledTextAreaBehaviorTest > org.fxmisc.richtext.model.StyledTextAreaBehaviorTest$ContextMenuTests.requestingContextMenuViaKeyboardWorksOnWindows
FAILED
    java.lang.AssertionError at StyledTextAreaBehaviorTest.java:86

Is there a setting I'm missing or something?

**Comment by JordanMartinez** [comment]
Hmm... That might just be a flaky test since the mouse version of it works fine.

**Comment by JordanMartinez** [comment]
I've looked into this issue more. It looks like TestFX does not properly create a `ContextMenuEvent` when the line `push(KeyEvent.CONTEXT_MENU)` is called. I added `println` statements to the area's `setOnKey[Pressed/Released/Typed]` handlers and only the released one was called. They're moving from using the internal API they use to create events to using `Event.fireEvent` to create such events. That migration will be complete in the next release I'm sure. So, for now, let's just ignore the test since the code does work as proven by the mouse event tests. We can enable it again when the TestFX issue is fixed.
